2021.5 Doc Edits (#353)
oneapi-src/oneDPL: documentation/library_guide/overview.rst

|onedpl_long| Overview
#######################################

> A parallel API can be used with the `C++ Standard Execution

Reviewer: ```suggestion
Parallel API can be used with the `C++ Standard Execution
```
I suggest to remove article as "Parallel API" is a name here

Policies <https://en.cppreference.com/w/cpp/algorithm/execution_policy_tag_t>`_

> to enable parallelism on your host.

Reviewer: ```suggestion
to enable parallelism on the host.
```
I suggest to keep "the host",  as it seems like a stable term in SYCL documentation.

#. To build with the |dpcpp_cpp|, see the `Get Started with the Intel® oneAPI DPC++/C++ Compiler
   <https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-dpcpp-compiler/top.html>`_
   for details.

> #. Set the environments for |onedpl_short| and |onetbb_short|.

Reviewer: ```suggestion
#. Set the environment variables for |onedpl_short| and |onetbb_short|.
```
I suggest this wording as in this case the environment should be common for oneDPL and oneTBB.
